packages/ripple-nuxt-tide/README.md

```diff
@@ -2,7 +2,180 @@
 
 > The Nuxt module for adding Tide integration.
 
-## Setup
+## Nuxt configuration
+
+A `nuxt.config.js` file should exist in the root of your nuxt-tide site.
+
+The basic anatomy of a `nuxt.config.js` file is:
+
+```js
+export default {
+  modules: [],
+  debug: false,
+  serverMiddleware: {},
+  head: {},
+  loading: {},
+  router: {},
+  styleResources: {},
+  ripple: {},
+  tide: {}
+}
+```
+
+### modules
+
+An array of all modules to include.
+See [Nuxt configuration modules](https://nuxtjs.org/api/configuration-modules).
+
+```js
+{
+  modules: [
+    '@dpc-sdp/ripple-nuxt-tide',
+    '@nuxtjs/style-resources',
+    // we want to always disallow bots on the example site
+    ['@nuxtjs/robots', {
+      UserAgent: '*',
+      Disallow: '/'
+    }]
+  ],
+}
+```
+
+### debug
+
+A boolean for setting the site in debug mode.
+
+### serverMiddleware
+
+An object that can include additional middleware such as basic authentication,
+or advanced redirects.
+
+```js
+{
+  serverMiddleware: [
+    '~/middleware/basic-auth'
+  ],
+}
+```
+
+### head
+
+An object that defines the headers of the page.
+See [Nuxt configuration head](https://nuxtjs.org/api/configuration-head).
+
+```js
+{
+  head: {
+    title: 'vic.gov.au',
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { name: 'msapplication-TileColor', content: '#da532c' },
+      { name: 'theme-color', content: '#ffffff' },
+      { hid: 'description', name: 'description', content: 'vic.gov.au' }
+    ],
+    link: [
+      { rel: 'apple-touch-icon', sizes: '180x180', href: '/favicon.ico' },
+      { rel: 'icon', type: 'image/png', sizes: '32x32', href: '/favicon-32x32.png' },
+      { rel: 'icon', type: 'image/png', sizes: '16x16', href: '/favicon-16x16.png' },
+      { rel: 'manifest', href: '/site.webmanifest' },
+      { rel: 'mask-icon', href: '/safari-pinned-tab.svg', color: '#5bbad5' }
+    ],
+    script: [
+      { src: '/js/gtag.js', body: true, defer: true },
+    ]
+  },
+}
+```
+
+### loading
+
+An object that takes a `color` property to customize the progress bar color.
+See [Nuxt configuration loading](https://nuxtjs.org/api/configuration-loading).
+
+```js
+{
+  loading: { color: '#0095EC' },
+}
+```
+
+### router
+
+An object that defines routes programmatically.
+See [Nuxt configuration router](https://nuxtjs.org/api/configuration-router) for
+further details.
+
+**Note:** Nuxt-tide adds its own routes which may conflict.
+
+```js
+{
+  router: {
+    extendRoutes (routes, resolve) {
+      routes.push({
+        path: '/custom-path',
+        alias: '/path-alias',
+        component: resolve(__dirname, 'modules/nuxt-tide/modules/profile/pages/search.vue')
+      })
+    }
+  },
+}
+```
+
+### styleResources
```

Comment:
This is added by `sass-resources-loader` in nuxt not by this module. So this is not the right place to doc it. I think it should be in the 'create-sdp-app'.

Reply:
Removed. It looks like https://github.com/dpc-sdp/ripple/pull/441 alters how this works, and this documentation isn't relevant anymore.